cluster/os-3.9.0-alpha.4/provider.sh

```diff
@@ -2,7 +2,7 @@
 
 set -e
 
-image="os-3.9@sha256:6d5e570e78981f7faf2deb6df2bb2f887a6ed2dd458b4178897ebe877316f8f8"
+image="os-3.9@sha256:c214267c1252e51f5ea845ac7868dbc219c63627e9f96ec30cc0a8e9e6e9fc0d"
```

Comment:
With this kind of reference we really loose insights of what changes in those images.

It's effectively pulling an arbitrary image.

What about making the image creation scripts part of this repo?

Reply:
It is actually the only way to make sure that everyone really uses the same docker image. It is not optimal but the best we have right now.

Reply:
Yes, I understand your motivation.

But it makes it pretty impossible to review requests like this.

I just wonder if it would make sense to just pull the image build scripts into this repository as well in order to include those changes (leading to the image change) in PRs.

Reply:
As a temporary solution, I can provider line on PR under our images repository